docs/app/partials/menu-link.tmpl.html

```diff
@@ -1,5 +1,5 @@
 <md-button ng-class="{'active' : isSelected()}"
-  ng-href="#{{section.url}}">
+  ng-href="#{{section.url}}" ng-click="itemClicked()">
```

Comment:
We can name it whatever we want. The purpose of `ng-click` here was to provide a hook differentiating link clicks from route changes. There wasn't a way to do that with `ng-href`.